src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html

```diff
@@ -1,157 +1,69 @@
-<h1> Student List</h1>
+<h1>Student List</h1>
+<p>
+  This page gives the student details for all your courses. Expand the cards below to view your students.
+  Use the <a routerLink="/web/instructor/search">search function</a> if you need to filter for a specific
+  student.
+</p>
+<hr/>
 
-<div class="card bg-light mb-4 pb-3">
-  <div class="card-body">
-    <div class="form-group">
-      <div class="input-group">
-        <input type="text" class="form-control" [(ngModel)]="searchQuery" (keyup.enter)="search()"
-               placeholder="Charlie Barnes">
-        <div class="input-group-append">
-          <button class="btn btn-primary" (click)="search()">Search</button>
+<ng-container *ngIf="courseTabList">
```

Comment:
There is no way that courseTabList can be undefined right?

Reply:
Yup updated